Clinical-Genomics/scout
# Description field for gene panels

Fix #1308 

- Panel description can be optionally provided when creating a new panel:

![image](https://user-images.githubusercontent.com/28093618/64602318-dc9c7980-d3be-11e9-9fad-248141db8482.png)

- Description can be alternatively modified or added from the panel page:

![image](https://user-images.githubusercontent.com/28093618/64602581-59c7ee80-d3bf-11e9-8cd1-35d68e86320a.png)

**How to test**
Install the branch on clinical-db stage:
```
ssh your.user@clinical-db.scilifelab.se
sudo -iu hiseq.clinical
cd servers/resources/clinical-dbscilifelab.se
bash update-scout-stage.sh panel_description
```

1. Try to create a new gene panel with and without description (tip: a csv file with a gene list is in the scout/demo data folder of scout)

1. Make sure that you can modify ( or cancel ) the panel description from the gene panel page.


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
